### DISCUSSION TITLE
Saved Search Advanced Filtering not working in more than one chrome session

### DISCUSSION THREAD

**Discussion by csilk** [burkybang/SuiteAdvanced-Feedback#195] [General]
I have multiple Chrome sessions/profiles, all with the plugin installed and authenticated.

The saved search filtering only works in the first Chrome profile where I installed the plugin, this is also the only chrome profile where I am logged into Chrome/Google (not sure if that part is important).

The below image is the feature I am referring to:

<img width="842" alt="Screenshot 2024-10-02 at 10 25 15" src="https://github.com/user-attachments/assets/7472ca82-76b2-4efb-bda9-df07502fc667">

The image below is how this looks in the other Chrome profiles.

<img width="473" alt="Screenshot 2024-10-02 at 10 30 40" src="https://github.com/user-attachments/assets/5b26009c-945a-4898-a033-cdbbef529c4f">

I'm not sure if this is a defect or if I'm doing something wrong?

**Comment by burkybang** [comment]
The screenshot is from a different Chrome extension unrelated to SuiteAdvanced.
